src/components/atoms/Checkbox/index.tsx

```diff
@@ -3,16 +3,21 @@ import { ICheckboxProps } from "../../../utils";
 import { Wrapper, HiddenCheckbox, CustomCheckbox, Label } from "./styles";
 import Check from "./Check";
 
-function Checkbox({ checked, label, disabled, ...rest }: ICheckboxProps) {
+/**
+ *
+ * Add label and name props to be a labeled checkbox
+ *
+ */
+function Checkbox({ checked, label, disabled, name, ...rest }: ICheckboxProps) {
   return (
-    <Wrapper checked={checked!} disabled={disabled!}>
+    <Wrapper checked={checked!} disabled={disabled!} htmlFor={name}>
```

Comment:
Why add ! at the end of checked? better to use a default value if the props are required